### PR TITLE
FIX - Changed deprecated lambda runtime 

### PR DESCRIPTION
**Pull request by Sizk**
*Issue #, if available:*
Terraform wouldn't deploy

*Description of changes:*
NodeJS14.x has been changed to NodeJS16.x

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by bfreiberg** [comment]
Node v16 is going to be deprecated soon as well. Would it be possible to update to v18 or even v20?

File: cloudfront-apigw-rest-api-dynamodb
[the remark this answers is not in the document]
Comment (Sizk) [reply cloudfront-apigw-rest-api-dynamodb:1]: No :/

[the remark this answers is not in the document]
Comment (Sizk) [reply cloudfront-apigw-rest-api-dynamodb:1]: fixed

**Comment by Sizk** [comment]
Done

**Comment by bfreiberg** [comment]
Thanks for the quick update. One thing I noticed during a test deployment: If the user is not using `us-east-1` an error is raised:

> Error: creating EC2 Subnet: InvalidParameterValue: Value (us-east-1a) for parameter availabilityZone is invalid. Subnets can currently only be created in the following availability zones: eu-central-1a, eu-central-1b, eu-central-1c.

Please include a line in the deployment instructions to adjust the value in `variables.tf` to their region.

**Comment by bfreiberg** [comment]
Looks good, thank you for your contribution

**Pull request merged**